src/mpcCoreKit.ts

```diff
@@ -807,6 +807,24 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return tssKey.toString("hex", FIELD_ELEMENT_HEX_LEN);
   }
 
+  /**
+   * WARNING: Use with caution. This will export the seed  for ed25519 KeyType.
+   *
+   * Exports the seed for the current account index.
```

Comment:
```suggestion
```

Sorry, overlooked this. Account indexes are not supported currently for ed25519.

Reply:
Removed that part of the comment.